test/test_utils.py

```diff
@@ -771,6 +776,32 @@ def test_arguments_are_printed_correctly_with_none_additional_message_specified(
         assert mock_click.echo.call_args_list == [
             call("arg 1: string option"),
             call("arg 2: 12"),
-            call("arg 3: {'key': 'value'}")
+            call("arg 3: {'key': 'value'}"),
         ]
-        mock_click.confirm.assert_called_once_with(confirmation_message, abort=True)
\ No newline at end of file
+        mock_click.confirm.assert_called_once_with(confirmation_message, abort=True)
+
+
+class TestWriteFilesToZip:
+    """Test class to test the write_files_to_zip functionality"""
+
+    def test_given_files_written_as_zip_file_to_given_location_with_correct_contents(
+        self, tmp_path
```

Comment:
I'm a little confused how this is working. There is no patch for `tmp_path`.

Reply:
So `tmp_path` is a fixture supplied by the pytest module (https://docs.pytest.org/en/stable/tmpdir.html), basically sets up a temporary folder for the test then cleans it up after completion.